### ISSUE TITLE
Why Multiplicity.SIMPLE edges cannot be included in vertex-centrix indices

### ISSUE THREAD

**Issue by zifnab87** [thinkaurelius/titan#1275]
I have this issue:
http://stackoverflow.com/questions/35833151/why-multiplicity-simple-edges-cannot-be-included-in-vertex-centric-indices

Is this on purpose?
Thanks,
Michail

**Comment by HonestManXin** [comment]
I totally agree with you. The titan community should loosen edge constraints.
So i think the constraint code in [Multiplicity.isConstrained](https://github.com/thinkaurelius/titan/blob/titan11/titan-core/src/main/java/com/thinkaurelius/titan/core/Multiplicity.java#L53)

```
public boolean isConstrained(Direction direction) {
        if (direction==Direction.BOTH) return isConstrained();
        if (this==MULTI) return false;
        if (this==SIMPLE) return true;
        return isUnique(direction);
    }
```

should be changed to 

```
public boolean isConstrained(Direction direction) {
        return isUnique(direction);
    }
```

That means we should constraint on edges with single in edge or out edge.
But it seems that the titan community is not active。So it will be a long time for this take effects.

**Comment by zifnab87** [comment]
@HonestManXin I am slowly preparing to migrate to Datastax DSE Graph when the Gremlin/Tinkerpop fluent APIs are ready.. I wonder if this issue was taken into account in that product too..
